doc/dev/dataplane_codegen_quick_start.md

```diff
@@ -0,0 +1,198 @@
+   license-header: MICROSOFT_MIT_NO_VERSION
+   clear-output-folder: true
+   no-namespace-folders: true
+   python: true
+   title: WebPubSubServiceClient
+   version-tolerant: true
+   package-version: 1.0.0b1
+   add-credential: true
+   credential-scopes: https://webpubsub.azure.com/.default
+   ```
+   ````
+
+   **Reviewers: which of the above can we remove?  We should keep only those needed for the "common" case. And if any could just be the default (e.g. license-header), we should remove those too. Then we should document whatever remains.**
+
+   There are some configuration in the template and you could reference [autorest/flags](https://github.com/Azure/autorest/blob/main/docs/generate/flags.md) to get more info. We need to replace the value in `package-name`, `title`,  `input-file`, `package-version`, `credential-scopes`, `output-folder` into **your own service's** `package-name`, `title`etc.
+
+3. **Run autorest to generate the SDK**
+
+   Now you can run this command in swagger folder you just created.
+
+   ```sh
+   autorest --version=latest --python --use=@autorest/python@latest  README.md
+   ```
+
+4. **Add necessary files under  ${PROJECT_ROOT}**
+
+   For a complete package, there are some other files you need to add:
+
+   - `CHANGELOG.md`: introduce the release history and important change for each release.
+
+   - `LICENSE`: license file
+
+   - `README.md`: import files which introduce what the service is and how to use this package
+
+   - `dev_requirements.txt`: claims some package which is needed to run test
+
+   - `sdk_packaging.toml`: configuration file to decide which file will be included in package.
+
+   - `setup.py`: most important files about dependency and supported version for Python.
+
+   Here is [template files](https://github.com/Azure/azure-sdk-for-python/tree/main/scripts/quickstart_tooling_llc/template), and you at least need to edit item (like `{{package_name}}`, `{{package_pprint_name}}`, etc) in `CHANGELOG.md`, `README.md`, `setup.py`, `sdk_packaging.toml` according to specific info of your own service.  It is easier to understand how to edit the files with reference existing service: [webpubsub](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/webpubsub/azure-messaging-webpubsubservice)
+
+   **Reviewers: I understand that work is in progress to generate all these files with a special "init" flag to autorest, so this item will probably go away in the near future.**
+
+## How to add convenience code
+
+**Reviewers: Somewhere, and here seems like as good a place as any, we need to tell teams how to add convenience / "grow up" code. There are certainly right and wrong ways to do this, so we need to give service teams some guidance on this. We could just give the basics here and then point to another document with full details, since most service teams won't bother to add convenience code, but we need the information here in case they do.**
```

Comment:
Yes, we were discussing with @iscai-msft we need this doc once she's done with MetricsAdvisor playing to understand all possible scenario.s